### PR TITLE
feat: ability to post with ctrl + enter

### PR DESCRIPTION
**Pull request by iMADi-ARCH**
## Related Issue

Issue #166 

## Description

This PR adds the functionality to post and comment with the almost universal hotkey `Ctrl + Enter` on windows and `Cmd + Enter` on mac os.

### CODE REVIEW

File: src/helper/isCtrlEnter.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { KeyboardEvent } from "react";
+
+/**
+ * Checks if given keyboard event is Ctrl + Enter or Cmd + Enter
+ * @param e Keydown event
+ * @returns true if Ctrl + Enter (on windows) or Cmd + Enter (on mac) is pressed
+ */
+export default function isCtrlEnter(e: KeyboardEvent<HTMLInputElement | HTMLTextAreaElement>) {
+  if ((e.ctrlKey || e.metaKey) && e.key === "Enter") {
```
Comment (Sanchitbajaj02) [inline src/helper/isCtrlEnter.ts:9]: is this handled for MacOS's command key?

**Review by Sanchitbajaj02 — APPROVED** [review]

File: src/helper/isCtrlEnter.ts (hunk 1 above)
Comment (iMADi-ARCH) [reply src/helper/isCtrlEnter.ts:9]: yep. metaKey is equivalent to cmd key in mac os

Comment (iMADi-ARCH) [reply src/helper/isCtrlEnter.ts:9]: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Comment (Sanchitbajaj02) [reply src/helper/isCtrlEnter.ts:9]: ok, so we are good to go

**Pull request merged**